server/eslint.config.mjs

```diff
@@ -61,4 +61,11 @@ export default defineConfig([{
             },
         }],
     },
-}]);
\ No newline at end of file
+
+    ignores: [
+        "target/**",
+        "dist/**",
+        "build/**",
+        "node_modules/**"
+    ]
```

Comment:
Adding this because otherwise it's now looking in these generated directories which it should not be doing.